examples/walkthrough/runner.sh

```
@@ -0,0 +1,99 @@
+# runner.sh — Reusable terminal UX framework for interactive walkthrough guides.
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)